### DISCUSSION TITLE
What workflow managers or engines are you using / supporting and what domain are you using them for?

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#60] [Discussion topics for #RSEAA2023]
For example, at WEHI, the Research Computing Platform is encouraging the use of NextFlow for Bioinformatics workflows.

What are other people using?

**Comment by supernord** [comment]
### Bioinformatics

- Nextflow https://www.nextflow.io/
- Galaxy https://galaxyproject.org/
- Snakemake https://snakemake.readthedocs.io/en/stable/
- No workflow language: code / scripts
- CWL https://www.commonwl.org/
- WDL https://github.com/openwdl/wdl

**Comment by alfski** [comment]
At WSU we still use a lot of shell & make scripts on our HPCs.
Had some training with NextFlow for Pawsey users in Astronomy area.

**Comment by pr4deepr** [comment]
Which workflow system would you recommend?

**Comment by hot007** [comment]
In regional environmental modelling in CSIRO we are using Nextflow and Airflow for model prep and local or remote HPC execution; Snakemake for routine data processing; and I believe there is some use of cronicle.

**Comment by rowlandm** [comment]
Has anyone been using Nix for package management?

**Reply by robmoss** [reply]
Only infrequently for personal use
